content/en/graphing/metrics/explorer.md

```diff
@@ -11,21 +11,56 @@ further_reading:
     text: "Metrics Distributions"
 ---
 
-Use the [Metrics Explorer page][1] to begin examining the [metrics][2] collected from your application by Datadog.
+## Overview
 
-Clicking into the "Graph" text box brings up a list of all the metrics that Datadog gathers from your application. You can also search the list by typing into the box. Selecting metrics populates real-time graph visualizations on the right hand side. By default, up to 20 graphs are shown. You can change this number by clicking on the "Options" gear.
+The [Metrics Explorer][1] is a basic interface for examining your metrics in Datadog. For more advanced options, create a [notebook][2], [screenboard][3], or [timeboard][4].
 
-{{< img src="graphing/metrics/explorer/explorer.png" alt="Metrics Explorer" responsive="true" style="width:60%;" >}}
+## Graphing
 
-"Over" allows you to narrow your results by tag. Under "One graph per", you can choose to split a metric into multiple graphs by host, container, region, team, and more.
+Click on the **Graph** text box to see a list of metrics you have sent to Datadog. Search the list by typing in the text box. Select a metric to populate a real-time graph visualization on the right side. Each metric you select creates a separate graph. Above the graphs, you can specify the time frame and graph size.
```

Comment:
Click on the **Graph** text box to see a list of metrics you have submitted to Datadog. To select a metric to populate a real-time graph visualization on the right side of the page, begin by typing in the text box and click to select. Each metric you select creates an individual graph. 

At the top of your graphs, you can specify the timeframe and graph size.

Reply:
Re-worded slightly.